docs/understanding-tracking-design/managing-your-tracking-scenarios/index.md

```diff
@@ -5,6 +5,7 @@ sidebar_position: 95
 sidebar_custom_props:
   offerings:
     - enterprise
+    - cloud
```

Comment:
Wouldn’t this mislead people into thinking the API is available for Cloud? Probably we need to move the offering definition to the child pages, where one would be enterprise + cloud and the other just enterprise. (The way I implemented this mechanism, it does not currently support defining something and then refining in a child page, although it’s of course possible to change.)

Reply:
Good catch again. I've altered the implementation, to be able to override the parent offerings. Please check if the implementation makes sense, my JS skills are straight from the first page of Google.

One side effect that I've noticed - is that child pages now got enterprise/cloud emoji assigned.

Old:
![image](https://github.com/snowplow/documentation/assets/126697986/3eccd377-0d36-4fd9-965d-a213d0b9b150)
New:
![image](https://github.com/snowplow/documentation/assets/126697986/8dd22095-1746-473d-8750-04d92345c373)

Let me know if anything needs to be done about the implementation/emojies

Reply:
@stanch Elisa says we are ready to merge this change, could you take a look?